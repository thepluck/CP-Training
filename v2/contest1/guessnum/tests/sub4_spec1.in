21
5926
6494
8059
9228
9867
28923
30109
31682
41137
41418
41665
44918
50673
57464
66010
67792
75203
81741
86636
91874
106756
