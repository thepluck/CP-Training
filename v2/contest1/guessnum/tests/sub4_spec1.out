7202686
8649395
13321089
17467833
19969999
171601501
185961654
205900019
347137002
351900395
356105872
413882285
526737856
677378695
893843615
942756569
1160138873
1370641665
1539712515
1731520424
2337918534
